app/policies/disbursement_policy.rb

```diff
@@ -7,16 +7,13 @@ def show?
 
   def new?
     user&.admin? || (
-      (record.destination_event.nil? || record.destination_event.users.include?(user)) &&
+      record.destination_event.nil? &&
```

Comment:
```suggestion
      record.destination_event.nil? &&
```
I don't think you want to be only approving disbursements with `nil` `destination_event`s

Reply:
I was very confused by the nil checks but it worked when I tested it so I left it.

Reply:
Were you signed in as an admin?

Reply:
I might've been.

Reply:
This policy still needs to be updated so that it's based on the source event's plan

Reply:
I think I just fixed what you said but not sure. I don't fully understand what the nil checks are for still.